src/Core/Silk.NET.BuildTools/PublicAPI/net5.0/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,840 @@
+#nullable enable
```

Comment:
Why are we tracking build tools as API?

Reply:
Just wasn't made automatically exempt. Fixed.